### ISSUE TITLE
Compilation issue with Yacc 3.0.2

### ISSUE THREAD

**Issue by arcanis** [aras-p/hlsl2glslfork#54]
```
arcanis@~/hlsl2glslfork/build # make
[  3%] Executing Bison on hlslang.y
hlslang.y:109.1-12: warning: deprecated directive, use ‘%pure-parser’ [-Wdeprecated]
 %pure_parser /* Just in case is called from multiple threads */
 ^^^^^^^^^^^^
[  6%] Executing flex on hlslang.l
Scanning dependencies of target hlsl2glsl
[  9%] Building CXX object CMakeFiles/hlsl2glsl.dir/hlslang/MachineIndependent/Gen_hlslang.cpp.o
[ 12%] Building CXX object CMakeFiles/hlsl2glsl.dir/hlslang/MachineIndependent/hlslang_tab.cpp.o
hlslang_tab.cpp: In function ‘int yyparse()’:
hlslang_tab.cpp:2223:30: error: too few arguments to function ‘int yylex(YYSTYPE*, void*)’
hlslang.y:105:16: note: declared here
hlslang.y:36:46: error: ‘parseContextLocal’ was not declared in this scope
hlslang.y:187:13: note: in expansion of macro ‘parseContext’
hlslang.y:36:46: error: ‘parseContextLocal’ was not declared in this scope
hlslang.y:196:17: note: in expansion of macro ‘parseContext’
...
```

And a lot more.

I guess it's because [YYPARSE_PARAM has been deprecated/removed](https://lists.gnu.org/archive/html/bison-patches/2012-06/msg00031.html).

**Comment by computerquip** [comment]
arcanis, you are correct. I have a pull request here: https://github.com/aras-p/hlsl2glslfork/pull/63

There are two open issues on this matter by the way. For reference: https://github.com/aras-p/hlsl2glslfork/issues/62

**Comment by computerquip** [comment]
This appears to be fixed with the merge of #63. Please close when convenient.

**Comment by aras-p** [comment]
Fixed by #63
